support/Environments/urania.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/env sh
+
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+spectre_setup_charm_paths() {
+    # Define Charm paths
+    export CHARM_ROOT=/u/guilara/charm_impi_2/mpi-linux-x86_64-smp
+    export PATH=$PATH:/u/guilara/charm_impi_2/mpi-linux-x86_64-smp/bin
```

Comment:
I suggest you create a module file like `/u/guilara/modules/charm/7.0.0-impi-2` that sets these paths. Then you can `module load` it below and don't have to call this extra `spectre_setup_charm_paths` function.

Reply:
I don't think I want to tackle this right now. Installing charm was a bit tricky

Reply:
Then better set these paths in `spectre_load_modules`, or else you always have to call this extra function?